### PR TITLE
fix: not and min breakpoints needed to be one pixel bigger to correctly exclude respective breakpoints

### PR DESCRIPTION
**Pull request by atomictangerine**

### CODE REVIEW

File: ui/theme/src/stitches.config.ts [hunk 1]
```diff
@@ -55,15 +55,15 @@ const WPDS = createStitches({
     lg: `(min-width: ${breakpoints.md}) and (max-width: calc(${breakpoints.lg} - 1px))`,
```
Comment (wp-aberg) [inline ui/theme/src/stitches.config.ts:55]: Can we remove all the -1px calcs? Otherwise we end up with a dead spot right at the breakpoint value.

Comment (wp-aberg) [reply ui/theme/src/stitches.config.ts:55]: https://github.com/washingtonpost/wpds-ui-kit/assets/102534985/ab911ece-6dcb-4cc4-b244-7d8644909bfe

**Review by ebgranger — CHANGES_REQUESTED** [review]
It looks like the layout goes bananas at 767px.

Screenshot 2023-06-27 at 12 21 32 PM

**Comment by ebgranger** [comment]
looked at BP page and a few components at all the different breakpoints and dragged the screen width back and forth on all previously mentioned pages. LGTM.

**Review by ebgranger — APPROVED** [review]

**Pull request merged**